corehq/apps/reports/standard/cases/basic.py

```diff
@@ -172,6 +179,9 @@ def case_owners(self):
                 demo=HQUserType.DEMO_USER in user_types,
                 commtrack=HQUserType.COMMTRACK in user_types,
             )
+            if HQUserType.DEACTIVATED in user_types:
```

Comment:
I thought this wasn't an option for the case list?  Isn't it just "all data" and "project data"?

Reply:
https://github.com/dimagi/commcare-hq/pull/21564/commits/5e033a668409f42280e067e43bfef5815d27dec8